anchorecli/cli/utils.py

```diff
@@ -974,6 +892,38 @@ def format_output(config, op, params, payload):
     return(ret)
 
 
+def format_vulnerabilities(payload, params):
+    obuf = ""
+    if 'query_type' not in params or not params['query_type']:
+        # payload will be a list with what is available as a query for the
+        # given image
+        for query in payload:
+            obuf += "%s: available\n" % query
+        return obuf + "\n"
+
+    if params['query_type'] in ['os', 'non-os', 'all']:
+        header = [
+            'Vulnerability ID', 'Package', 'Severity', 'Fix', 'CVE Refs',
+            'Vulnerability URL', 'Type', 'Feed Source', 'Package Path'
```

Comment:
Instead of "Feed Source", I'd like to call this "Feed Group". I think that makes more sense as it matches the API data.

Reply:
addressed in the latest push